src/server.ts

```diff
@@ -1,4 +1,4 @@
-import  express, {  Application, NextFunction, Request, Response } from 'express';
+import  express, {  Application, Request, Response } from 'express';
 import routes from './routes/messages';
 
 const app: Application = express();
```

Comment:
Connect your express app to squelize

Reply:
Added sequelize and get table users from db.